specs/src/specs/namespace.md

```diff
@@ -57,30 +57,32 @@ The ID is encoded as a byte slice of length 28.
 
 ## Reserved Namespaces
```

Comment:
I rephrased this section a bit for clarity. Please LMK if you think I should revert anything @staheri14